docs/source/markdown/podman-build.1.md

```diff
@@ -54,6 +54,10 @@ Add a custom host-to-IP mapping (host:ip)
 Add a line to /etc/hosts. The format is hostname:ip. The **--add-host** option
 can be set multiple times.
 
+#### **--all-platforms**
+
+Instead of building for a set of platforms specified using the **--platform** option, inspect the build's base images, and build for all of the platforms for which they are all available.  Stages that use *scratch* as a starting point can not be inspected, so at least one non-*scratch* stage must be present for detection to work usefully.
```

Comment:
I'm struggling with the first sentence.  I think this is better, but I'm not sure it's right or that I like it a lot.  Maybe "found" instead of "available" in my suggestion?

```suggestion
Instead of building for a set of platforms specified using the **--platform** option, inspect the build's base images, and build for all of the platforms that are available.  Stages that use *scratch* as a starting point can not be inspected, so at least one non-*scratch* stage must be present for detection to work usefully.
```

Reply:
I can fix this here but this is copy paste from buildah-bud so someone should fix it there as well.